.github/workflows/deprecated-update-snapshot.yml

```diff
@@ -1,4 +1,7 @@
-name: Update snapshots
+# This workflow is deprecated. Use ./update-snapshots.yml instead.
```

Comment:
Keeping the old workflow around until I've proven that the updated one works properly.

Some aspects of the rewrite are impossible to test before merging this PR. I tried out the label-deletion logic in a throaway repo, so I have reasonable confidence, but still.